Tools/Toolkit/Sources/ToolkitLibrary/Models/Language.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  MIT License
+//  Copyright (c) 2020-2021 Raycast. All rights reserved.
+//
+
+import Foundation
+
+enum Language {
+  case applescript
+  case bash
+  case python
+  case ruby
+  case swift
+  case custom(String)
+
+  init(_ rawValue: String) {
+    let value = rawValue.lowercased()
+
+    switch value {
+    case "applescript", "osascript":
+      self = .applescript
+    case "bash", "zsh":
+      self = .bash
+    case "python", "python2", "python3":
+      self = .python
+    case "ruby":
+      self = .ruby
+    case "swift":
+      self = .swift
+    default:
+      self = .custom(value)
+    }
+  }
+
+  var icon: String? {
+    switch self {
+    case .applescript:
+      return "icon-applescript.png"
+    case .bash:
+      return "icon-bash.png"
+    case .python:
+      return "icon-python.png"
+    case .ruby:
+      return "icon-ruby.png"
+    case .swift:
+      return "icon-swift.png"
+    default:
+      return nil
+    }
+  }
+
+  var socialName: String {
```

Comment:
`socialName`? I would call it `displayName`, what do you think?
```suggestion
  var displayName: String {
```

Reply:
Great suggestion! I’m terrible for names... Am I the only one? 🤔

Reply:
I was just confused that it's some social media thing :D

Reply:
See? I'm terrible for names! hahaha leading confusion to anyone else's mind since 1986! 🤣